stacks/catalog/vault-playground.yaml

```diff
@@ -0,0 +1,19 @@
+import: []
+vars:
+  stage: playground
```

Comment:
stage is a required variable, see https://atmos.tools/quick-start/provision#stack-search-algorithm